tests/test_n26_importer.py

```diff
@@ -277,3 +282,66 @@ def test_raise_on_payee_in_multiple_accounts(language):
                 ],
             },
         )
+
+
+def test_extract_conversion(importer, filename):
+    with open(filename, 'wb') as fd:
+        fd.write(
+            _format(
+                '''
+                {header}
+                "2022-08-01","Alice","{iban_number}","Income","Muster GmbH","Income","56.78","","",""
+                "2022-08-02","Bob","{iban_number}","Outgoing Transfer","Home food","Foo","-42.0","","",""
+                "2022-08-03","Charlie","{iban_number}","Outgoing Transfer in a foreign currency","Foreign food","Bar","-10.0","9.13","CHF","0.9687"
```

Comment:
I'm curious why `-10.0` and `9.13` in this example don't have the same sign. They're outgoing expenses, right?

Reply:
When you convert funds using Transferwise, the `Amount (Foreign Currency)` does hold a positive amount